nss-day-cohort-48/rare-client-boastful-bulldogs
# User Profile Detail Page

Under user list you can now view a user's profile detail when clicking on their name.

## Steps to Test
(fetch server too)
1. git fetch --all
2. git checkout anm-userprofiledetail
3. go to user list and click on a name
4. verify you can see the user's profile detail
5. also verify steve's shows as admin when looking at profile type
